ui/components/common/CommonToolbar.js

```diff
@@ -50,6 +52,8 @@ class CommonToolbar extends Component {
                         type="text"
                         placeholder="Filter by Name"
                         className={toolbarFilter}
+                        value={get(this.props.filter, "name", "")}
```

Comment:
I think you can avoid installing the new package for this, perhaps destructuring:
```
const { onFilter, filter: { name } } = this.props;
```

Reply:
I understand that. I'm not very experienced with JavaScript, but I think that this is particularly very confusing for someone coming from other languages.